src/libcore/iter.rs

```diff
@@ -660,6 +665,145 @@ pub trait IteratorExt: Iterator + Sized {
         None
     }
 
+    /// Return the index of the last element satisfying the specified predicate
+    ///
+    /// If no element matches, None is returned.
+    #[inline]
+    #[stable]
+    fn rposition<P>(&mut self, mut predicate: P) -> Option<uint> where
```

Comment:
I'm personally not a huge fan of the `where`-by-itself (I like the next line), but oh well!